smac/facade/smac_facade.py

```diff
@@ -340,16 +340,71 @@ def optimize(self):
         return incumbent
 
     def get_tae_runner(self):
+        '''
+            returns target algorithm evaluator (TAE) object
+            which can run the target algorithm given a
+            configuration
+
+            Returns
+            -------
+            smac.tae.execute_ta_run.ExecuteTARun
+        '''
         return self.solver.intensifier.tae_runner
 
     def get_runhistory(self):
+        '''
+            returns the runhistory 
+            (i.e., all evaluated configurations and the results)
+
+            Returns
+            -------
+            smac.runhistory.runhistory.RunHistory
+        '''
         if not hasattr(self, 'runhistory'):
             raise ValueError('SMAC was not fitted yet. Call optimize() prior '
                              'to accessing the runhistory.')
         return self.runhistory
 
     def get_trajectory(self):
+        '''
+            returns the trajectory 
+            (i.e., all incumbent configurations over time)
+
+            Returns
+            -------
+            List of entries with the following fields: 
+            'train_perf', 'incumbent_id', 'incumbent',
+            'ta_runs', 'ta_time_used', 'wallclock_time'
+        '''
+
         if not hasattr(self, 'trajectory'):
             raise ValueError('SMAC was not fitted yet. Call optimize() prior '
                              'to accessing the runhistory.')
         return self.trajectory
+
+    def get_X_y(self):
```

Comment:
In the beginning, I thought so, too.
However, the rh2epm methods do a lot more at the moment than we would like to do here:
- it imputes censored data -- we don't want this here, but a vector indicating whether some y's are censored
- it uses a log-transformation for runtime scenarios -- I would need to undo it again
- it uses the unit-hypercube parameter representation -- I would also need to undo it, since Stefan preferred the original values

Since, the undoing of all these things needs at least as many code lines as this function, I chose to write it directly without reusing rh2epm.